apps/web/src/components/tx/SignOrExecuteForm/SignOrExecuteForm.tsx

```diff
@@ -112,8 +112,12 @@ export const SignOrExecuteForm = ({
         !!signer?.isSafe,
         customOrigin,
       )
+
+      if (customOrigin !== props.origin) {
+        trackAddNote()
+      }
```

Comment:
What happens if the user didn't add a note and its not a safe apps transaction. Are both values undefined and we would try to track something?

Reply:
Just trying to understand why we initialize `customOrigin` with `props.origin` instead of just leaving it undefined and if it exists we track an event.

Reply:
undefined equals undefined though.

Reply:
Ah, simply to avoid doing `customOrigin || props.origin` when passing it down.